include/vcpkg/base/messages.h

```diff
@@ -1117,6 +1140,7 @@ namespace vcpkg
         (msg::system_name, msg::value),
         "'{value}' is the linkage type vcpkg would did not understand. (Correct values would be static ofr dynamic)",
         "Invalid {system_name} linkage type: [{value}]");
+    DECLARE_MESSAGE(InvalidOptionForRemove, (), "", "'remove' accepts either libraries or '--outdated'");
```

Comment:
This should have a note for localizers that 'remove' is a command that should not be changed.